m4_sessions/1811_inning/timprov.md

```diff
@@ -26,6 +34,11 @@
 
 ## Session Outline (Agenda for session, include timing for each session and who is facilitating)
 
+Intro
+Whole group: Pass the clap; One word story;
+Split into two groups: Kumja; Family portrait; Hitchhiker
+Split into four groups: Count to 20; Bus Stop; Freeze
+Whole group: Reflection - "How do I use this in the workplace"?
```

Comment:
I would recommend giving everyone 1 minute to reflect on their own and then maybe start in groups of 3 and then go whole group.  You will get more voices and ideas!